### ISSUE TITLE
No games launch with DXVK

### ISSUE THREAD

**Issue by beemdvp** [ValveSoftware/Proton#2698]
# Compatibility Report
All games that require DXVK

## System Information
- GPU: R9 290X
- Driver/LLVM version: Mesa: 18.2.0-rc3 AMD RADV HAWAII (LLVM 8.0.0)
- Kernel version: 5.1
- Proton version: 4.2-4

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

Proton log with Grim Dawn: https://gist.github.com/ibrahimpmahmood/0d8d9ae62d190d8989c9e3757863e99c

Running `lspci -k`
```
01:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Hawaii XT / Grenada XT [Radeon R9 290X/390X]
	Subsystem: Gigabyte Technology Co., Ltd R9 290X WindForce 3X OC
	Kernel driver in use: amdgpu
	Kernel modules: radeon, amdgpu
```
Installed on steam and proton same HDD drive as linux (EXT4)

Vulkaninfo log: https://gist.github.com/ibrahimpmahmood/c80d34f2fb7d094ad27fe21f22a61cb4

## Symptoms 
No games that require DXVK seem to work. Works fine when I enable d3d9 but never d3d11

**Comment by doitsujin** [comment]
Since you're running a Hawaii card, checkout these instructions on how to get Vulkan up and running:
https://github.com/ValveSoftware/Proton/wiki/For-AMD-users-having-issues-with-non-OpenGL-games

If that still doesn't fix it, make sure that you have the required Vulkan packages installed.

**Comment by beemdvp** [comment]
I have already done this, made no difference unfortunately :/ and I have made sure that all vulkan packages have been installed. I have Path of Exile installed using lutris natively and it detects DXVK just fine. Anything on Steam play however doesnt run unless its d3d9

**Comment by kisak-valve** [comment]
Hello @ibrahimpmahmood, does moving `/usr/share/vulkan/icd.d/intel*.json` somewhere out of the way make a difference? Does any vulkan application like `vulkaninfo` work?

**Comment by beemdvp** [comment]
Hey @kisak-valve Ive moved out the intel files just now to no avail. `vulkaninfo` command does work for me too

Edit: I have updated the issue with vulkaninfo log and lspci -k log

**Comment by jarrard** [comment]
I have once had a similar issue as this, and a complete proton/steamplay wipe resolved the problem.

Disable proton/steamplay, then uninstall proton from tools then delete any proton folders in the steamapps common location (and also on any ntfs steam folders your using). 

Be sure to delete any compatdata folders also, restart steam (and pc perhaps), re-enable proton and try again.

**Comment by beemdvp** [comment]
Hey @jarrard, I had to be more extreme in this case. I installed the newest distro to 19.04 which by default would has the latest kernel followed by the latest mesa drivers which I think was the issue. My mesa driver was too old to boot anything despite having the latest 19.2 packages but it wasnt completely installing it...Was very odd, thank you for trying to help! Everything works fine after upgrading the actual distro :+1:
